Loop.xcodeproj/project.pbxproj

```diff
@@ -3461,6 +3461,7 @@
 				CODE_SIGN_STYLE = "$(LOOP_CODE_SIGN_STYLE)";
 				DEVELOPMENT_TEAM = "$(LOOP_DEVELOPMENT_TEAM)";
 				INFOPLIST_FILE = Loop/Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
```

Comment:
Please push this setting up to the project settings (not target specific).

Reply:
Sure. I only updated the necessary targets, but I can update for the project instead.